tslint.json

```diff
@@ -15,6 +15,14 @@
     "no-relative-imports": false,
     "no-stateless-class": false,
     "trailing-comma": [true, {"multiline": "always", "singleline": "never"}],
+    "no-for-in-array": false,
+    "restrict-plus-operands": false,
```

Comment:
Not if I want to use it in vscode, we had a discussion about that before and the result is that both these rules don't add much value in the end and it's better to have vscode be able to show these.